src/client/java/teammates/client/scripts/helper/DataMigrationForSanitizedDataHelper.java

```diff
@@ -0,0 +1,41 @@
+package teammates.client.scripts.helper;
+
+/**
+ * Utility class that provides methods to help with migrating data to its desanitized form.
+ * Contains a private constructor to prevent instantiation of the class.
+ */
+public final class DataMigrationForSanitizedDataHelper {
+
+    private DataMigrationForSanitizedDataHelper() {
+        // utility class
+    }
+
+    /**
+     * Returns true if the {@code string} has evidence of having been sanitized.
+     * A string is considered sanitized if it does not contain any of the chars '<', '>', '/', '\"', '\'',
+     * and contains at least one of their sanitized equivalents or the sanitized equivalent of '&'.
+     */
+    public static boolean isSanitizedHtml(String string) {
+        if (string == null) {
+            return false;
+        }
+        if (isTextContainingAny(string, "<", ">", "\"", "/", "\'")) {
+            return false;
+        }
+        return isTextContainingAny(string, "&lt;", "&gt;", "&quot;", "&#x2f;", "&#39;", "&amp;");
+    }
+
+    /**
+     * Returns true if text contains at least one of the {@code strings},
+     * false if text does not contain any of the strings or if no strings are given.
```

Comment:
I wanted to be more clear on what happens when no strings are given but I removed it since it is quite clearly implied by the first part of the sentence.